ansible/roles/content-snapshot-indexer/templates/scripts/run-script.j2

```diff
@@ -4,12 +4,13 @@
 #$2 populate_all_data: true/false - Especially for UserCache indexer
 #$3 identifier: userid- Especially for UserCache indexer script to cache specific user id details
 #$4 date: YYYY-MM-DD- Especially for UserCache indexer script to cache all the user data from this specific updated date.
+#$5 populate_anonymous_user: true/false - Especially for UserCacheIndexer to read anonymous user's data
 
-echo "Executing the script $1 $2 $3 " 
+echo "Executing the script $1 $2 $3 $5"
 case "$1" in
    "USER_CACHE_INDEXER") 
       echo "Invoked RedisUserDataIndexer"
-      nohup {{ content_snapshot_path }}/scripts/RedisUserDataIndexer.sh $2 $3 & 
+      nohup {{ content_snapshot_path }}/scripts/RedisUserDataIndexer.sh $2 $3 $5&
```

Comment:
Where is $4?

Reply:
I have made the changes by removing populate_all_data so now there are 4 arguments